.github/workflows/cd.yaml

```diff
@@ -0,0 +1,88 @@
+name: cd
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  unit-testing:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: niden/actions-memcached@v7
```

Comment:
Nice to add a comment explaining why this is needed.

Reply:
This is copy-paste from flux's existing build workflow